db/schema.rb

```diff
@@ -10,571 +10,571 @@
-    t.string   "collection_id"
-    t.string   "role"
-    t.string   "local_path"
-    t.string   "alt_text"
-    t.string   "target_url"
-    t.integer  "height"
-    t.integer  "width"
-    t.datetime "created_at",    null: false
-    t.datetime "updated_at",    null: false
-  end
-
-  create_table "collection_type_participants", force: :cascade, options: "ENGINE=InnoDB DEFAULT CHARSET=latin1" do |t|
-    t.integer  "hyrax_collection_type_id"
-    t.string   "agent_type"
-    t.string   "agent_id"
-    t.string   "access"
-    t.datetime "created_at",               null: false
-    t.datetime "updated_at",               null: false
-    t.index ["hyrax_collection_type_id"], name: "hyrax_collection_type_id", using: :btree
-  end
-
-  create_table "content_blocks", force: :cascade, options: "ENGINE=InnoDB DEFAULT CHARSET=latin1" do |t|
-    t.string   "name"
-    t.text     "value",        limit: 65535
-    t.datetime "created_at",                 null: false
-    t.datetime "updated_at",                 null: false
-    t.string   "external_key"
-  end
-
-  create_table "curation_concerns_operations", force: :cascade, options: "ENGINE=InnoDB DEFAULT CHARSET=latin1" do |t|
-    t.string   "status"
-    t.string   "operation_type"
-    t.string   "job_class"
-    t.string   "job_id"
-    t.string   "type"
-    t.text     "message",        limit: 65535
-    t.integer  "user_id"
-    t.integer  "parent_id"
-    t.integer  "lft",                                      null: false
-    t.integer  "rgt",                                      null: false
-    t.integer  "depth",                        default: 0, null: false
-    t.integer  "children_count",               default: 0, null: false
-    t.datetime "created_at",                               null: false
-    t.datetime "updated_at",                               null: false
-    t.index ["lft"], name: "index_curation_concerns_operations_on_lft", using: :btree
-    t.index ["parent_id"], name: "index_curation_concerns_operations_on_parent_id", using: :btree
-    t.index ["rgt"], name: "index_curation_concerns_operations_on_rgt", using: :btree
-    t.index ["user_id"], name: "index_curation_concerns_operations_on_user_id", using: :btree
-  end
-
-  create_table "featured_works", force: :cascade, options: "ENGINE=InnoDB DEFAULT CHARSET=latin1" do |t|
-    t.integer  "order",      default: 5
-    t.string   "work_id"
-    t.datetime "created_at",             null: false
-    t.datetime "updated_at",             null: false
-    t.index ["order"], name: "index_featured_works_on_order", using: :btree
-    t.index ["work_id"], name: "index_featured_works_on_work_id", using: :btree
-  end
-
-  create_table "file_download_stats", force: :cascade, options: "ENGINE=InnoDB DEFAULT CHARSET=latin1" do |t|
+ActiveRecord::Schema.define(version: 2018_05_29_220525) do
+
+  create_table "bookmarks", id: :integer, options: "ENGINE=InnoDB DEFAULT CHARSET=utf8", force: :cascade do |t|
```

Comment:
I'm wondering how db/schema.rb got so many changes, particularly the `CHARSET` it seems dangerous to just suddenly have our schema dump switching character sets if our database hasn't gone through a proper migration.
`config/database.yml` and `config/mysql/mysqld.cnf` seem to disagree on this as well. This might be good to double check with the live database to see what character set it's using now. imo we should be using utf8, but I'd rather not corrupt data.

Reply:
Yeah im not entirely sure where this change came, but i assume its part of the reason why i had to drop the database and rebuild the schema. I can look deeper.